app/assets/stylesheets/user_index.scss

```diff
@@ -1,10 +1,29 @@
+@mixin shadow1 {
+  box-shadow: 1px 1px 1px #888888;
```

Comment:
Color literals like `#888888` should only be used in variable declarations; they should be referred to via variable everywhere else.